mapbox_tilesets/utils.py

```diff
@@ -71,7 +71,17 @@ def validate_tileset_id(tileset_id):
     return re.match(pattern, tileset_id, flags=re.IGNORECASE)
 
 
-def validate_geojson(feature):
+def validate_linear_ring(count, geometry):
+    if geometry["type"] == "Polygon":
+        coord = geometry["coordinates"]
+        is_ring = all([elem[0] == elem[-1] for elem in coord])
```

Comment:
From https://www.rfc-editor.org/rfc/rfc7946#section-3.1.6, this should also validate >= 4 coordinates. Also, what about MultiPolygon, FeatureCollection, etc.?

Rather than try to build out our existing simple validation, can we rely on a library to do this?

Reply:
Good point, updated in the two most recent commits!